wg-notes/fxl-a11y/index.html

```diff
@@ -980,12 +980,10 @@ <h3>Alternative rendering methods</h3>
 
             <p>Alternative rendering methods for ebook content are methods of rendering beyond the visual, specifically auditory and tactile rendering (i.e., screen reader output or braille). While fixed layout formatting is often used to represent highly visual content, not all reading systems support visual display, and when visual display is available, screen sizes may be limited. Therefore, many reading systems might avoid supporting fixed layout content due to concerns about a poor reading experience. In addition to concerns about rendering, it must also be considered that many users face difficulties consuming and navigating complex fixed layout content. Alternative rendering methods should be considered as a replacement or addition to fixed layout rendering.</p>
 
-            <p>If the fixed layout publication does not conform to the recommendations made in this document, or WCAG [[wcag2]], providing alternate renderings of the content may result in an unusable or poor reading experience. Content that is not formatted in conformance with WCAG may result in output that has an incorrect reading order, broken sentences, or choppy pronounciation. The user should be informed if the content does not have accessibility metadata that would provide clarity on whether alternate renderings are supported, such as <code>dcterms:conformsTo</code> with a value for WCAG, or metadata values that conform to properties like <a href="https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/principles/#supports-nonvisual-reading">Supports nonvisual reading</a> from the <a href="https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/principles/">User Experience Guide for Displaying Accessibility Metadata</a>.</p>
+            <p>If the fixed layout publication does not conform to the recommendations made in this document, or WCAG [[wcag2]], providing alternate renderings of the content may result in an unusable or poor reading experience. Content that is not formatted in conformance with WCAG may result in output that has an incorrect reading order, broken sentences, or choppy pronounciation. The user should be informed if the content does not have accessibility metadata that would provide clarity on whether alternate renderings are supported, such as <code>dcterms:conformsTo</code> with a value for WCAG, or metadata values that conform to properties like <a href="https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/principles/#supports-nonvisual-reading">Supports nonvisual reading</a> from the <a href="https://www.w3.org/community/reports/publishingcg/CG-FINAL-a11y-display-guidelines-20250422/">User Experience Guide for Displaying Accessibility Metadata</a>.</p>
```

Comment:
Same changes as previous comment.
```suggestion
            <p>If the fixed layout publication does not conform to the recommendations made in this document, or WCAG [[wcag2]], providing alternate renderings of the content may result in an unusable or poor reading experience. Content that is not formatted in conformance with WCAG may result in output that has an incorrect reading order, broken sentences, or choppy pronounciation. The user should be informed if the content does not have accessibility metadata that would provide clarity on whether alternate renderings are supported, such as <code>dcterms:conformsTo</code> with a value for WCAG, or metadata values that conform to properties like <a href="https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/principles/#supports-nonvisual-reading">Supports nonvisual reading</a> from the <a href="https://www.w3.org/publishing/a11y/metadata-display-guide/guidelines/">Accessibility Metadata Display Guide for Digital Publications</a>.</p>
```